flutter/integration_test/expected_accuracy.dart

```diff
@@ -0,0 +1,50 @@
+class Interval {
+  final double min;
+  final double max;
+
+  const Interval({required this.min, required this.max});
+}
+
+const Map<String, Interval> _imageClassification = {
+  'CPU': Interval(min: 1.0, max: 1.0),
+  'NNAPI': Interval(min: 0.8999999761581421, max: 0.8999999761581421),
```

Comment:
The number can just have only 2 or 3 decimal places to make it easy to read, since we use an interval right now.

Reply:
Done.
I reduces the precision to 4 decimal places since this is what we already use in the UI.